Assignments/Program.cs

```diff
@@ -1 +1,46 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T13 - Strong Password
+// ---------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T15 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Check for Isogram Word:-" + "\x1B[0m");
+      Console.WriteLine ("An isogram is a word that has no duplicate letters.");
+      string output;
+      List<string> testCases = new () { "apple", "banana", "cat", "doctrine", "elephant", "fruits", "pebble", "rubiks", "yellow", "zebra" };
+      Console.WriteLine ();
+      Console.WriteLine ("\x1B[4m" + "Test Cases:" + "\x1B[0m");
+      for (int i = 0; i < testCases.Count; i++) {
+         Console.Write ($"{i + 1}. {testCases[i]} - ");
+         (output, Console.ForegroundColor) = IsIsogram (testCases[i]) ? ("Isogram", Green) : ("Not an Isogram", Red);
+         Console.WriteLine ($"{output}");
+         Console.ResetColor ();
+      }
+      Console.Write ("\nNow, let's enter a word: ");
+      string input = Console.ReadLine ().ToUpper ();
+      (output, Console.ForegroundColor) = IsIsogram (input) ? ("Isogram", Green) : ("Not an Isogram", Red);

```

Comment:
This and next two lines are repeated for test case stuff too.
Always move same code to a separate method and use it.
Here, you could just pass the string to the method which can check if it is isogram or not and print accordingly. Otherwise, for eg if you are changing the color to Yellow from Green then you have to change in two places.